network/network.test.js

```diff
@@ -19,34 +19,88 @@ import { renderHook, act } from '@testing-library/react-hooks';
 import { useNetworkStatus } from './';
 
 describe('useNetworkStatus', () => {
+  const map = {};
+
+  const ectStatusListeners = {
+    addEventListener: jest.fn().mockImplementation((event, callback) => {
+      map[event] = callback;
+    }),
+    removeEventListener: jest.fn()
+  };
+
+  afterEach(() => {
+    Object.values(ectStatusListeners).forEach(listener => listener.mockClear());
+  });
+
+  // Tests that addEventListener or removeEventListener was called during the lifecycle of the useEffect hook within useNetworkStatus
```

Comment:
Can we split this comment across two lines? I would break prior to lifecycle.

Reply:
Will do